### ISSUE TITLE
Subscribe to calendar feeds for organizations and groups via Google Cal/iCal

### ISSUE THREAD

**Issue by andrewtavis** [activist-org/activist#716]
### Terms

- [X] I have searched [open and closed feature requests](https://github.com/activist-org/activist/issues?q=is%3Aissue+label%3Afeature)
- [X] I agree to follow activist's [Code of Conduct](https://github.com/activist-org/activist/blob/main/.github/CODE_OF_CONDUCT.md)

### Description

This issue is a placeholder issue for the idea that revolves around subscribing to events from organizations via an existing calendar application. There would be various ways of doing this depending on the calendar application, but exporting the events in a calendar formatted file would be a great first step. This could be done for iCal for Mac/iOS users, Google Calendar, and other calendar applications could be added from there :)

Note that this issue is related to #715 where we're discussing exporting the data in a more structured format, whereas this issue is exporting files that can be imported into specific applications.

### Contribution

Happy to discuss and work on this later as we make more progress on the events API and other features 😊

**Comment by kiran7902** [comment]
Is this feature still blocked?

**Comment by andrewtavis** [comment]
I think for the time being it is as we need to finalize the UI, but that should be done in the coming weeks. Let's keep an eye on this, @kiran7902, and thanks for your offer to help 😊

**Comment by Nishthajain7** [comment]
Is this issue still blocked? I’d like to start working on it if it’s not.

**Comment by andrewtavis** [comment]
Thanks for writing in here, @Nishthajain7! I think we should be good to tentatively look into this issue as we do have events in both organizations and groups now. Would you want to set up the project and confirm that you can see the events in organizations via following the `View organizations` button on the development home screen and selecting events in the sidebar? From there, what data should be added to the calendar entry? Time of the event definitely, name of the event and location? I guess that would be enough to get started :)

Let us know on the above! I'll assign you, and happy to support as needed 😊

**Comment by Nishthajain7** [comment]
I have already completed the setup for this project and yes, I can see the events. Would u want the events be exported in a .ics formatted file?

**Comment by andrewtavis** [comment]
That would make the most sense, @Nishthajain7! Thanks for the suggestion :) By all means proceed and let us know if further questions come up.

**Comment by andrewtavis** [comment]
Where would you think to put the option? At the top right of the events pages for organizations and groups?

**Comment by Nishthajain7** [comment]
![Image](https://github.com/user-attachments/assets/c74a9162-9ab9-4320-a385-a5809914c828)
We can have a subscribe option  along with the share option or as you said, top right of an event

**Comment by andrewtavis** [comment]
That would be good for getting the individual events, and maybe we could have something that exports all events for an organization or group in a single file? That could be a further PR after you send along the work for adding the option to the tooltip :)

**Comment by Nishthajain7** [comment]
I have Added `Subscribe` buttons for **individual events** on both - the tooltip and on the about page of the event.
Also implemented the backend to export all events of an org. Where would you like to have the option for subscribing to all events of an org

**Comment by andrewtavis** [comment]
Yes, I think this would be great, @Nishthajain7! Thanks so much! 😊 What do you think about all events that are upcoming? Might be a bit of a weird experience if the user is getting a bunch of past events 🤔

**Comment by Nishthajain7** [comment]
We can have another filter on the events page, along with the `filter by topic` dropdown.

**Comment by andrewtavis** [comment]
Closed by #1276 :) Thanks for the initial work towards getting this up and running, @Nishthajain7! Please let us know if there's another issue you'd have interest in 😊
